src/sentry/static/sentry/app/components/events/interfaces/frame.jsx

```diff
@@ -32,6 +33,7 @@ const Frame = createReactClass({
     emptySourceNotation: PropTypes.bool,
     isOnlyFrame: PropTypes.bool,
     timesRepeated: PropTypes.number,
+    registers: PropTypes.objectOf(PropTypes.string.isRequired),
```

Comment:
does `isRequired` do anything as an arg to `objectOf`?

Reply:
afaik if you don't have it you can have nulls as object values, but tbh I didn't check.